### DISCUSSION TITLE
Wildcard: "Send to" or "Copy"

### DISCUSSION THREAD

**Discussion by unoriginalscreenname** [adieyal/sd-dynamic-prompts#351] [Ideas]
It would be great to include an option to send the wildcard to the txt2img prompt or negative space and include the proper formatting.  Or maybe even a "copy wildcard" button that wraps it in the {} for the prompt.  I know this sounds trivial, but I find myself copying the __Prompt__ and then having to do the brackets and then paste it in the middle.  It would be a small, but nice enhancement to just copy it with proper formatting!

**Comment by akx** [comment]
You don't need the brackets in the prompt, that's only for variants.

I.e. if you have a wildcard `animals`, you can refer to it with `__animals__` alone (and that's indeed what you see in the top right corner of the wildcard browser when you select an entry, so no need to wrap it in the double underscores by hand).

If you wanted the prompt to choose between animals and shapes, then you'd need `{__animals__|__shapes__}`.

**Reply by unoriginalscreenname** [reply]
Ah, didn't realize that.  A "copy to clipboard" button might be nice still!
